src/ontology/mondo-edit.obo

```diff
@@ -274763,11 +274763,11 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0013772
-name: congenital cataract-hearing loss-severe developmental delay syndrome
+name: Huppke-Brendel syndrome
 subset: ordo_disease {source="Orphanet:300313"}
 synonym: "CCHLND" RELATED ABBREVIATION [MONDO:Lexical, OMIM:614482]
-synonym: "congenital cataract-deafness-severe developmental delay syndrome" EXACT [Orphanet:300313]
-synonym: "congenital cataracts, hearing loss, and neurodegeneration" RELATED [MONDO:Lexical, OMIM:614482]
+synonym: "congenital cataracts, hearing loss, and neurodegeneration" EXACT [MONDO:Lexical, OMIM:614482]
+synonym: "congenital cataracts, hearing loss, and neurodegeneration" EXACT [OMIM:614482]
```

Comment:
What is the difference between `EXACT [MONDO:Lexical, OMIM:614482]` and `EXACT [OMIM:614482]`?

Reply:
you are right, these should be combined into one line